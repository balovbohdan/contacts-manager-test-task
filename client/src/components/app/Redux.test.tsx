import {shallow} from 'enzyme';
import * as React from 'react';

import {Redux} from './Redux';

describe('Redux connector', () => {
    it('Should match snapshot', () => {
        const redux = shallow(<Redux>Foo</Redux>);

        expect(redux).toMatchSnapshot();
    });
});