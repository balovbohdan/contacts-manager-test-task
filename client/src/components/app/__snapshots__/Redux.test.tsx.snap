// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Redux connector Should match snapshot 1`] = `
<Provider
  store={
    Object {
      "dispatch": [Function],
      "getState": [Function],
      "replaceReducer": [Function],
      "subscribe": [Function],
      Symbol(observable): [Function],
    }
  }
>
  Foo
</Provider>
`;
